packages/@okta/vuepress-site/use_cases/inline_hooks/saml_hook/saml_hook/index.md

```diff
@@ -0,0 +1,404 @@
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "doobie"
+            }
+          ]
+        },
+        "array": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "Array 1"
+            },
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "Array2"
+            },
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "Array3"
+            }
+          ]
+        },
+        "middle": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "middellllll"
+            }
+          ]
+        },
+        "firstAndLast": {
+          "attributes": {
+            "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:unspecified"
+          },
+          "attributeValues": [
+            {
+              "attributes": {
+                "xsi:type": "xs:string"
+              },
+              "value": "7d6a50c8-4d7e-4058-9c5b-2cc98cecd294"
+            }
+          ]
+        }
+      },
+      "lifetime": {
+        "expiration": 300
+      }
+    }
+  },
+  "eventTypeVersion": "1.0",
+  "cloudEventVersion": "0.1",
+  "eventType": "com.okta.saml.tokens.transform",
+  "contentType": "application/json"
+}
+```
+
+## Sample Listing of JSON Payload of Response
+
+```json
+{
+   "commands": [
+    {
+      "type": "com.okta.assertion.patch",
+      "value": [
+        {
+          "op": "replace",
+          "path": "/claims/array/attributeValues/1/value",
+          "value": "replacementValue"
+        },
+        {
+          "op": "replace",
+          "path": "/authentication/authnContext",
+          "value": {
+            "authnContextClassRef": "Something:different?"
+          }
+        },
+        {
+          "op": "add",
+          "path": "/claims/foo",
+          "value": {
+            "attributes": {
+              "NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:basic"
+            },
+            "attributeValues": [
+              {
+                "attributes": {
+                  "xsi:type": "xs:string"
+                },
+                "value": "barer"
+              }
+            ]
+          }
+        },
+        {
+          "op": "replace",
+          "path": "/conditions/audienceRestriction",
```

Comment:
Let's remove it, we will soon restrict the ability to change audience.

Reply:
Removed.